carbon-design-system/carbon
# fix(a11y): fixing UI shell windows high contrast mode accessibility

Closes #6897 

Fixing UI shell windows high contrast mode accessibility issues

#### Changelog

**Changed**

UI Shell component

#### Testing / Reviewing

On a Windows 10 machine, go to `settings --> ease of access --> high contrast` and flick the switch. Compare the storybook in Edge (Should be in High Contrast) to Chrome (Should look normal). 
For **Firefox**, you'll need to open it up and type `about:config` into the address bar. Then, search for `layout.css.prefers-contrast` and set it to `true`.

- Visit [UI Shell](http://localhost:9000/?path=/story/ui-shell--fixed-side-nav) component
- Verify that the carets show in HCM
- Verify the focus state works all elements
